frame/assets/src/tests/traits.rs

```diff
@@ -320,7 +320,7 @@ mod multicurrency {
 
 		#[test]
 		fn test_minimum_balance_implementation(
-			account in accounts(),
+			_account in accounts(),
```

Comment:
same here, why is this changing?

Reply:
It is not used and the linter is complaining and suggesting the _ for unused variables. I guess it should be removed if it is not intended to be used?

Reply:
I have removed it altogether , it is unused